Cargo.toml

```diff
@@ -39,6 +39,7 @@ members = [
   "crates/turborepo",
   "crates/turborepo-ffi",
   "crates/turborepo-lib",
+  "crates/turborepo-scm",
```

Comment:
Have the Rust folks already though how we're going to divide up crates? Asked differently... why is this not in `turborepo-lib`?

Reply:
`turborepo-lib` is the code for the front-end of turbo, i.e. the entry part of the Go sandwich. We could in theory put this in `turborepo-lib` and import it into `turborepo-ffi` but we'd be relying on Rust to optimize away the two copies of `turborepo-lib` (one from the `turbo` crate and one from `turborepo-ffi`).

I do think this is worth discussing further. More crates means in theory faster build times because Rust's compilation unit is a crate (as opposed to other languages where it's a single file). But we will pay the price with the orphan rule, which is basically a restriction around trait implementation. Fortunately for us, our code is not very trait-y, so we may be able to avoid that.

Reply:
> But we will pay the price with the orphan rule, which is basically a restriction around trait implementation. Fortunately for us, our code is not very trait-y, so we may be able to avoid that.

I hope that one day in the very near future I understand what this means :)

> I do think this is worth discussing further.

Sounds good, I'll follow along the discussion in the internal channel!

> `turborepo-lib` is the...entry part of the Go sandwich. 

Gotcha, sounds like it could be named something a bit more obvious then